token-lending/js/src/instructions/flashLoan.ts

```diff
@@ -0,0 +1,54 @@
+import { TOKEN_PROGRAM_ID } from '@solana/spl-token';
+import { PublicKey, SYSVAR_CLOCK_PUBKEY, TransactionInstruction } from '@solana/web3.js';
```

Comment:
Getting a warning that `SYSVAR_CLOCK_PUBKEY` isn't used.

Reply:
My bad, missed it.